WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -221,7 +221,11 @@ + (NSArray *)routes
 {
   FBElementCache *elementCache = request.session.elementCache;
   XCUIElement *element = [elementCache elementForUUID:request.parameters[@"uuid"]];
-
+  
+  //if element is not specified then use root element i.e. application
+  if(element == nil) {
```

Comment:
Yeah `/tap/uuid` is legacy from iOS-driver that we plan to kill. Where are up for adding new end points however be compliant to WD spec https://w3c.github.io/webdriver/webdriver-spec.html